verticapy/connect.py

```diff
@@ -142,25 +132,98 @@ def change_auto_connection(name: str):
         f.close()
     else:
         raise NameError(
-            "The input name is incorrect. The connection '{}' has never been created.\nUse the new_auto_connection function to create a new connection.".format(
+            "The input name is incorrect. The connection '{}' has never been created.\nUse the new_connection function to create a new connection.".format(
                 name
             )
         )
 
+# ---#
+def close_connection():
+    """
+---------------------------------------------------------------------------
+Close the Database connection.
+    """
+    if verticapy.options["connection"]["conn"]:
+        try:
+            verticapy.options["connection"]["conn"].close()
+        except:
+            pass
+
+# ---#
+def connect(section: str, dsn: str = ""):
+    """
+---------------------------------------------------------------------------
+Connect to the Database.
+
+Parameters
+----------
+section: str / connector
+    Name of the section in the configuration file.
+dsn: str, optional
+    Path to the file containing the credentials. If empty, the 
+    VERTICAPY_CONNECTIONS environment variable will be used.
+    """
+    prev_conn = verticapy.options["connection"]["conn"]
+    verticapy.options["connection"]["conn"] = vertica_conn(section, dsn)
+    verticapy.options["connection"]["dsn"] = dsn
+    verticapy.options["connection"]["section"] = section
+    if prev_conn:
+        try:
+            prev_conn.close()
+        except:
+            pass
```

Comment:
Usually, it is not a good practice to pass the exceptions. It the function silently fails to accomplish its task without any complaint.